cves/kernel/CVE-2019-9162.yml

```diff
@@ -266,8 +275,12 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    This vulnerability did not include anything related to sandbox features. The
```

Comment:
Which sandbox features? Be more specific here.

Reply:
Updated